### PR TITLE
[PIL-1385-pillar2] Update pillar2-frontend accessibility statement

### PR DESCRIPTION
**Pull request by ridow-hmrc**
Update accessibility statement

### CODE REVIEW

**Comment by ridow-hmrc** [comment]
[Accessibility statement for Report Pillar 2 top-up taxes service – GOV.UK.pdf](https://github.com/user-attachments/files/17778026/Accessibility.statement.for.Report.Pillar.2.top-up.taxes.service.GOV.UK.pdf)

**Comment by JamesMMiller** [comment]
build error, looks like we need to update the tests:
```
[info] - should enforce WCAG version consistency for pillar2-frontend *** FAILED ***
[info]   The visible label for the link "Change: Can we contact the primary and secondary contact by telephone?" does not fully match its accessible name. This doesn’t meet WCAG 2.1 success criteria 2.5.3  (label in name). "2.1" did not start with substring "2.2" (ServicesISpec.scala:232)
```

**Review by JamesMMiller — APPROVED** [review]

**Review by oscarduignan — APPROVED** [review]
the pillar2 team have requested write access, but they've added their team approval - so rather than block this until the write access request is completed, I've agreed with them to approve and merge for them

**Pull request merged**
